### DISCUSSION TITLE
Questions about the flythrough demo "robot factory room¨

### DISCUSSION THREAD

**Discussion by AlverGant** [NVlabs/instant-ngp#317] [Q&A]
Congratulations for your work it is really awesome in terms of performance and quality!

My setup is working OK no issues here.

In order to build a fly-through we only have to compute a room sized nerf and afterwards use the camera path editor? Or there are more details to it?

Best regards

**Comment by Tom94** [comment]
Correct, you can use the camera path editor to design the path of your flythrough and you can then use the python bindings to traverse the camera path (and render images as you go) to your liking.
